### PR TITLE
[Frontend] "Données personnelles" + "Mentions légales" pages

### PR DESCRIPTION
**Pull request by Destinyyyy**
### Description
~~Draft, awaiting for confirmation of the page's content from David~~
Content updated

Implementation of pages:
- Données personnelles
- Mentions légales

### To confirm
- "Mentions légales" content OK? (It is a copy/paste from https://pass.sports.gouv.fr/mentions-legales/, but excluding the "Enquete" section)
- "Données personnelles" content OK? (The numbering seems OFF)

### Screenshot desktop + mobile données personnelles
desktop
mobile

### Screenshot desktop + mobile mentions légales
desktop
mobile

### CODE REVIEW

File: site/src/app/v2/mentions-legales/page.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  return (
+    <div>
+      <PageHeader
+        title="Mentions légales"
+        subtitle=""
+        classes={{
+          container: styles['page-header'],
+        }}
+      />
+      <div className={styles.wrapper}>
```
Comment (maxime-chaillet) [inline site/src/app/v2/mentions-legales/page.tsx:16]: mettre un <main> à la place du div

File: site/components/PageHeader/PageHeader.tsx [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+  classes?: {
+    container?: string;
+  };
 }
 
-export default function PageHeader({ title, subtitle }: IProps) {
+export default function PageHeader({ title, subtitle, classes }: IProps) {
   return (
-    <div className={styles.container}>
+    <div className={cn(styles.container, classes?.container)}>
```
Comment (maxime-chaillet) [inline site/components/PageHeader/PageHeader.tsx:14]: mettre un `<header>` à la place du div

File: site/src/app/v2/politique-de-confidentialite/page.tsx [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+  return (
+    <div>
+      <PageHeader
+        title="Politique de confidentialité"
+        subtitle="Politique de confidentialité relative au traitement de données personnelles réalisé dans le cadre de la délivrance du pass Sport 2024"
+        classes={{
+          container: styles['page-header'],
+        }}
+      />
+      <div className={styles.wrapper}>
```
Comment (maxime-chaillet) [inline site/src/app/v2/politique-de-confidentialite/page.tsx:17]: mettre un `main` ici à la place du div

File: site/src/app/v2/mentions-legales/page.tsx (hunk 1 above)
Comment (Destinyyyy) [reply site/src/app/v2/mentions-legales/page.tsx]: https://github.com/betagouv/pass-sport/pull/64/commits/21d608ee92d82cc7776c46245e0c0326aeca37ef

File: site/components/PageHeader/PageHeader.tsx (hunk 2 above)
Comment (Destinyyyy) [reply site/components/PageHeader/PageHeader.tsx]: https://github.com/betagouv/pass-sport/pull/64/commits/21d608ee92d82cc7776c46245e0c0326aeca37ef

File: site/src/app/v2/politique-de-confidentialite/page.tsx (hunk 3 above)
Comment (Destinyyyy) [reply site/src/app/v2/politique-de-confidentialite/page.tsx]: https://github.com/betagouv/pass-sport/pull/64/commits/675100144e76c9968135e6cbdc4ab8277cdc1d1d

**Pull request merged**